my-entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
```

Comment:
I can't see any evidence you actually use this entrypoint file, can we delete it?